app/Actions/DisplayHelpScreen.php

```diff
@@ -18,20 +16,23 @@ class DisplayHelpScreen
 
     public function __invoke()
     {
+        $consoleWriter = app('console-writer');
```

Comment:
Could we inject this once you make it a class

Reply:
Done.